src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -10,9 +10,9 @@ function(add_generalized_harmonic_executable
     "EvolutionMetavars<${INITIAL_DATA}, ${BOUNDARY_CONDITIONS}>"
     "${LIBS_TO_LINK}"
     )
-endfunction(add_generalized_harmonic_executable)
+endfunction(add_generalized_harmonic_executable_with_horizon)
 
-function(add_generalized_harmonic_no_horizon_executable
+function(add_generalized_harmonic_executable
```

Comment:
I think I would prefer that the name explicitly says that there is no horizon, and therefore people don't need to remember whether the generic name "generalized_harmonic_executable" has a horizon.  So perhaps `add_generalized_harmonic_executable_without_horizon`?